src/Components/Nav/CommunityNav/CommunityNav.scss

```
@@ -0,0 +1,182 @@
+@import "../../../Styles/commons.scss";
+@import "../../../Styles/reset.scss";
+@import url("https://fonts.googleapis.com/css2?family=Nanum+Gothic:wght@400;700;800&display=swap");
+
+%flex-space-between {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.communityNav {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  width: 100%;
+  height: 130px;
+  margin: auto;
+  background-color: white;
+  box-sizing: border-box;
+  box-shadow: 0 4px 20px -8px darkgray;
+
+  font-family: "Nanum Gothic", sans-serif;
+
+  .searchSection {
+    @extend %flex-space-between;
+    width: 90%;
+    margin: 10px auto 0px auto;
+
+    .navMenuWrapper {
+      display: flex;
+      align-items: center;
+      width: 200px;
+      .logo {
+        width: 40px;
+        height: 40px;
+        margin-right: 20px;
+        cursor: pointer;
+      }
+
+      .menuWrapper {
+        display: flex;
+        width: 100px;
+        span {
+          margin: 0px 10px;
+          font-family: "Noto Sans KR", sans-serif;
+          font-size: 14px;
+          font-weight: bolder;
+          cursor: pointer;
+          &:hover {
+            opacity: 0.5;
+          }
+        }
+      }
+    }
+
+    .navSearchBarWrapper {
+      @extend %flex-space-between;
+      margin: auto;
+
+      .searchBar {
+        position: relative;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        width: 680px;
+        height: 40px;
+        padding: 0px 5px;
+        background-color: #f7f8f9;
+        border: 0.5px solid #e9edeb;
+        border-radius: 22px;
+
+        .searchBarPlaceHolder {
+          display: flex;
+          justify-content: center;
+          align-items: center;
+
+          .searchIcon {
+            width: 18px;
+            height: 18px;
+            margin-right: 10px;
+            opacity: 0.8;
+          }
+          span {
+            color: #1b1d1f;
+            font-size: 13px;
+            font-weight: 400;
+          }
+        }
+
+        .searchInput {
+          position: absolute;
+          right: 0;
+          left: 0;
+          width: 680px;
+          height: 40px;
+          padding-right: 5px;
+          padding-left: 5px;
+          border: 0;
+          border-radius: 22px;
+          background-color: transparent;
+          text-align: center;
+          &:focus {
+            outline: none;
+          }
```

클릭했을 때 테두리가 생기는 것을 방지할 목적이라면 
focus일 때 말고 그냥 바로 속성을 추가해주세요.
```suggestion
          
            outline: none;
          
```

---

네!